Sources/Foundation/FileManager+POSIX.swift

```diff
@@ -471,6 +471,18 @@ extension FileManager {
         return self.string(withFileSystemRepresentation: buf, length: Int(len))
     }
     
+    internal func _recursiveDestinationOfSymbolicLink(atPath path: String) throws -> String {
+        let _resolvedPath = try _fileSystemRepresentation(withPath: path) {
+            realpath($0, nil)
+        }
+        guard let resolvedPath = _resolvedPath else {
+            throw _NSErrorWithErrno(errno, reading: true, path: path)
+        }
+
+        let resolvedPathString = String(cString: resolvedPath)
```

Comment:
This could just be `return String(cString: resolvedPath)`

Reply:
Done.